src/main/resources/assets/computercraft/lua/rom/programs/echo.lua

```diff
@@ -0,0 +1 @@
+print( ... )
```

Comment:
The point still stands though - shell scripting isn't a thing and so it doesn't really serve much purpose.